src/components/Dashboard/Form/DocumentForm.js

```diff
@@ -1,6 +1,7 @@
 import React, { Component } from "react";
 import FormHeader from "./FormHeader/FormHeader.js";
 import { Form } from "react-final-form";
+import arrayMutators from 'final-form-arrays'
```

Comment:
no semicolon at the end of the line? :P

Reply:
Hmm in `Files changed` semicolon is on the right place. Probably comments to outdated commit or GitHub makes jokes.

Reply:
weird, I clicked on pending pull requests, my bad, then there was no comment here.